src/messageschannel.cpp

```diff
@@ -110,7 +142,7 @@ QString MatrixMessagesChannel::sendMessageCallback(const Tp::MessagePartList &me
     QString content;
     for (auto &part : messageParts) {
         // TODO handle other parts?
-        if(part.count(QStringLiteral("content-type")) &&
+        if (part.count(QStringLiteral("content-type")) &&
```

Comment:
We don't need to know the *count* of the values. Please use `contains` instead.
Also, it is OK to have a line up to 120 characters in length, so you can reformat the condition in less lines for better readability.